scripts/20-kansas.sh

```diff
@@ -32,6 +32,13 @@ ogr2ogr -sql "SELECT '2016' AS year, 'Kansas' AS state, 'Sedgwick' as county, PR
         -t_srs EPSG:4326 -f GPKG -nln state -append \
         $TEMPDIR/state.gpkg data/20-kansas/2016/20173-sedgwick/precincts.geojson
 
+#
+# Add Shawnee County (FIPS 227) to the statewide Geopackage file.
+#
+ogr2ogr -sql "SELECT '2016' AS year, 'Kansas' AS state, 'Shawnee' as county, NAME AS precinct, 'polygon' AS accuracy FROM OGRGeoJSON" \
```

Comment:
`20` instead of `Kansas` for state, please

Reply:
`20227` instead of `Shawnee` for county, please

Reply:
And for "precinct" it should probably be concat of `20227` and `PrecinctIdentifier` (which is the ID versus the name that has the identifier postfix).

Reply:
To be clear, the rest of the counties use `Kansas` for state and the county name, not FIPS, for county. I should change all of them, not just Shawnee?

Reply:
For precinct, there seems to be a mix in the other KS counties, but a preference toward using the VTD code. That's part of `PRECINCTID` in Shawnee, so I'll use that. Unless you want the name and VTD concat?